widget/textgrid.go

```diff
@@ -47,9 +47,15 @@ type TextGridStyle interface {
 	BackgroundColor() color.Color
 }
 
+// TextGridTextStyle defines the text style that can be applied to a TextGrid cell.
+type TextGridTextStyle interface {
```

Comment:
Because the `TextGridTextStyle` must always be a `TextGridStyle` should it extend I to bake it into the API?

p.s. It's missing the `// Since: 2.5` comment